pweave/scripts.py

```diff
@@ -50,6 +62,12 @@ def weave():
     if options.figformat is not None:
         opts_dict["figformat"] = ('.%s' % options.figformat)
 
+    if opts_dict.pop('verbosity'):
+        logging.getLogger("ipykernel").propagate = True
+        ipykernel_handler = logging.StreamHandler()
+        logging.getLogger("ipykernel").addHandler(ipykernel_handler)
```

Comment:
line should be split into two (with ipykernel_logger intermediate variable)

Reply:
It's 4 lines. you are highlighting

Reply:
```suggestion
        ipykernel_logger = logging.getLogger("ipykernel")
        ipykernel_logger.propagate = False
```